common/src/ethereum.rs

```diff
@@ -132,13 +141,16 @@ pub enum WalletError {
     Local(#[from] ethers::signers::WalletError),
     #[error("no wallet specified")]
     NoWallet,
+    #[error(transparent)]
+    CallError(#[from] walletconnect::client::CallError),
```

Comment:
Move above `NoWallet` and call it `WalletConnect` to match the other variants

Reply:
https://github.com/radicle-dev/radicle-cli/commit/6bd4b556fdbdb58f9af097f15e2af011db40ed5d#diff-799f6adcb3a8a0469e7950f80331d4ffcb715140d420d1f4b25bad88ee61adc9R144-R145